packages/curve-ui-kit/src/widgets/Footer/Link.tsx

```diff
@@ -9,7 +11,16 @@ export type LinkProps = {
 }
 
 export const Link = ({ label, href, icon, target = '_blank' }: LinkProps) => (
-  <Button component="a" href={href} target={target} rel="noreferrer" color="ghost" variant="link" startIcon={icon}>
+  <Button
+    component={href.startsWith('http') ? LinkMui : RouterLink}
+    to={href}
+    href={href}
```

Comment:
doesn't this generate errors in the console, as the invalid `to` prop will be passed to the `a` element?
might be worth to implement a separate component with the 'http' logic

Reply:
Didn't give an error but I'll replace it with spreading property objects depending on the conditional